integration/daemon/daemon_test.go

```diff
@@ -182,7 +183,9 @@ func TestConfigDaemonSeccompProfiles(t *testing.T) {
 func TestDaemonProxy(t *testing.T) {
 	skip.If(t, runtime.GOOS == "windows", "cannot start multiple daemons on windows")
 	skip.If(t, os.Getenv("DOCKER_ROOTLESS") != "", "cannot connect to localhost proxy in rootless environment")
-	ctx := testutil.StartSpan(baseContext, t)
+
+	// Don't setup OTEL here to avoid it hitting the HTTP proxy.
+	ctx := context.Background()
```

Comment:
Curious; do we configure OTEL to be using HTTP? Or are we not configuring it to disable the "discovery" (`localhost:<some port>`)?

ISTR our intent was to disable the automatic discovery on localhost, but if it's still doing that, we may have missed something (or regressed somewhere)

Reply:
Yes we use the OTEL http endpoint in CI: https://github.com/moby/moby/blob/9d1541526c5beab51d2344b47624a160f199bbe6/.github/actions/setup-tracing/action.yml#L13

The intent here is to not start any spans specifically for this test, even if OTLP exporter is configured.